tests/test_start_date.py

```diff
@@ -13,25 +14,29 @@ def test_run(self):
         """
         Testing that the tap respects the start date
         - INCREMENTAL
-            - Verify 1st sync (start date=today-N days) record count > 2nd sync (start date=today) record count.
-            - Verify minimum bookmark sent to the target for incremental streams >= start date for both syncs.
-            - Verify by primary key values, that all records in the 2nd sync are included in the 1st sync since 2nd sync has a later start date.
+            - Verify 1st sync (start date=today-N days) record count > 2nd sync
+                (start date=today) record count.
+            - Verify minimum bookmark sent to the target for incremental streams >= start date
+                for both syncs.
+            - Verify by primary key values, that all records in the 2nd sync are included in
+                the 1st sync since 2nd sync has a later start date.
         - FULL TABLE
             - Verify that the 2nd sync includes the same number of records as the 1st sync.
-            - Verify by primary key values, that the 2nd sync and 1st sync replicated the same records.
+            - Verify by primary key values, that the 2nd sync and 1st sync replicated the same
+                records.
         """
 
-        start_date_1_epoch = self.dt_to_ts(self.first_start_date)
-        start_date_2_epoch = self.dt_to_ts(self.second_start_date)
+        start_date_1_epoch = self.dt_to_ts(self.first_start_date, self.START_DATE_FORMAT)
+        start_date_2_epoch = self.dt_to_ts(self.second_start_date, self.START_DATE_FORMAT)
 
         ##########################################################################
-        ### Update Start Date for 1st sync
+        # Update Start Date for 1st sync
         ##########################################################################
 
         self.START_DATE = self.first_start_date
```

Comment:
here, as well, no need to make `START_DATE` variable class specific. Name it as `START_DATE`